pages/common/cloc.md

```diff
@@ -14,3 +14,7 @@
 - Compare 2 directory structures and count the differences between them:
 
 `cloc --diff {{/directory/one}} {{/directory/two}}`
+
+- Ignore files that are ignored by VCS, such as git:
+
+`cloc --vcs git {{/path/to/directory}}`
```

Comment:
Remove preceding `/`

Reply:
The leading / is consistent with the existing examples in the file.

Reply:
The file is quite old, that's why. It definitely needs to be updated according to the newer standards.

Reply:
Yes, let's change all of the examples while we are here.

Reply:
Got it. I've removed the leading / from all the paths in this file.

Reply:
@agnivade, I am going to resolve this review due all the leading `/` have been removed of the paths in this file. Excuse me the interference.

Reply:
I tried to approve this to merge this PR, but still seems to appear this issue as a pending change request by @agnivade. Anyone knows why and what could be the solution?

Reply:
Yes, unless you are an org member and a pending request is there, you cannot force push.

Reply:
All right! Thank you for this information, @agnivade. Nice to know it for the next times with the same case.